test/e2e_node/services/apiserver.go

```diff
@@ -96,9 +104,15 @@ func (a *APIServer) Name() string {
 }
 
 func getAPIServerClientURL() string {
-	return apiserverClientURL
+	return framework.TestContext.Host
 }
 
 func getAPIServerHealthCheckURL() string {
-	return apiserverHealthCheckURL
+	return framework.TestContext.Host + "/healthz"
+}
+
+func generateTokenFile(tokenFilePath string) error {
+	klog.Info("DEBUG: Generating token")
+	tokenFile := fmt.Sprintf("%s,kubelet,uid,system:masters\n", framework.TestContext.BearerToken)
```

Comment:
@tallclair Is it okay to use the built-in group `system:masters` here? Do we need to use a new group with limited privileges?

Reply:
The privileged group is fine I think.